components/mediators/datamapper/org.wso2.carbon.mediator.datamapper.engine/src/main/java/org/wso2/carbon/mediator/datamapper/engine/core/mapper/MappingResource.java

```diff
@@ -69,6 +75,10 @@ public JSFunction getFunction() {
         return function;
     }
 
+    public List getPropertiesList() {
```

Comment:
It's always better to add a method comment for public methods, even though it is a getter for properties list. We can mentioned which properties etc.